tutor/commands/config.py

```diff
@@ -220,8 +224,31 @@ def patches_list(context: Context) -> None:
     renderer.print_patches_locations()
 
 
+@click.command(name="edit", help="Edit config.yml of the current environment")
+@click.pass_obj
+def edit(context: Context) -> None:
+    config_file = tutor_config.config_path(context.root)
+
+    if not os.path.isfile(config_file):
+        raise exceptions.TutorError(f"Missing config file at {config_file}")
+
+    open_cmd = None
+
+    if which("open"):  # MacOS & linux distributions that ship `open`. eg., Ubuntu
+        open_cmd = ["open", config_file]
+    elif which("xdg-open"):  # Linux
+        open_cmd = ["xdg-open", config_file]
+    elif which("start"):  # Windows
+        open_cmd = ["start", '""', config_file]
+    else:
+        raise exceptions.TutorError(f"Failed to find utility to launch an editor.")
```

Comment:
In the error message, we can specify which utilities are used by the command internally so that user can install them on the system if needed (not all the users of Tutor are dev, some are operators who might not want to delve into code to see what is happening under the hood).

Reply:
After some research, it seems to me that Windows will always have `start`, macOS will always have `open`, and Linux desktop environments will always have `xdg-open`. That leaves only non-desktop Linux users and perhaps BSD users. I think those users will know their systems better then we will, and they will be capable of figuring out how to open a yaml file as long as we tell them where it is 😄  @tecoholic , rather than letting them know what to install, could you just enhance this error message a bit just to let users know that they can edit configuration at `{config_file}`?